source/docs/software/wpilib-tools/glass/index.rst

```diff
@@ -0,0 +1,12 @@
+Glass
+=====
+Glass is a new dashboard and robot data visualization tool. Its GUI is extremely similar to that of the :ref:`Simulation GUI <docs/software/wpilib-tools/robot-simulation/simulation-gui:Simulation User Interface>`. In its current state, it is meant to be used as a programmer's tool rather than a proper dashboard in a competition environment.
```

Comment:
We should add a note stating that its not offered in the last of dashboards in the NI DS